src/web_app/src/pages/drl_hyperParams.tsx

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import React, {useEffect} from "react";
+import GlobalStyles from "@mui/material/GlobalStyles";
+import CssBaseline from "@mui/material/CssBaseline";
+import {AppbarStyle} from "../components/appBar";
+import Container from "@mui/material/Container";
+import logging from "../config/logging";
+import {useNavigate, useLocation} from "react-router-dom";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import TextField from "@mui/material/TextField";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import {Types} from "../types/data";
+import axios from "axios";
+import Backdrop from "@mui/material/Backdrop";
+import sleep from "../config/sleep_funct";
+import CircularProgress from "@mui/material/CircularProgress";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+
+export function DrlHyperParamsPage() {
+    const url = useLocation().pathname;
+    const dataset_choice = url.split("/")[2];
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    const navigate = useNavigate();
+    const [
+        open,
+        setOpen
+    ] = React.useState(false);
+    const [
+        override_check,
+        setOverride_check
+    ] = React.useState(false);
+    const [
+        nb_epochs,
+        setNb_epochs
+    ] = React.useState<number>(10000);
+    const [
+        learning_rate,
+        setLearning_rate
+    ] = React.useState<number>(0.9);
+    const [
+        batch_size,
+        setBatch_size
+    ] = React.useState<number>(32);
+    const [
+        momemtum,
+        setMomentum
+    ] = React.useState<number>(0.2);
```

Comment:
It won't be easy with what I'm currently doing on this page. In fact your suggestion should work if I use a form component but that is not what I did. In your case, when the user clicks, we save all the current values in the form, I understood. I will modify the code later to use the form and do it like that.

Reply:
It is possible to only set on value at a time with this method : 

```js
setMesValeurs({
        ...mesValeurs,
        valeur1: 'Je suis une fraise',
    })
}
```

The `...` notation before a JSON-like object destructurise it (in the same way `**` would do in python) then we override a specific value (here value1) this lead to an object that is the same as previously but with only one value changed. From what I understand, the ability to change one value at a time is critical to make this change easy. So with this small trick would it be still hard ? If it would, then I think that it might be a better idea to let this as is.